readme.md

```diff
@@ -2,7 +2,7 @@
 
 ![screenshot](https://raw.githubusercontent.com/jlord/patchwork/gh-pages/patchwork-ss.png)
 
-This repository is a website hosted by [GitHub Pages](http://pages.github.com). It goes along with [Git-it](http://www.github.com/jlord/git-it-electron), a desktop application for learning Git and GitHub.
+This repository iss a website hosted by [GitHub Pages](http://pages.github.com). It goes along with [Git-it](http://www.github.com/jlord/git-it-electron), a desktop application for learning Git and GitHub.
```

Comment:
Why are you changing the "is" into "iss"? That is a mispelling, and I do not see any other point of this PR.